### ISSUE TITLE
Storm Peaks quest -  [The Drakkensryd] - not working chain for Sons of Hodir

### ISSUE THREAD

**Issue by stish7** [dalaranwow/dalaran-wow#4961]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. The quest start from Gretta the Arbiter it requires to use Hyldsmeet Photo-Drake and  go to the top of  Temple of Storms where around it must be flying Photo-Drake's with Hyldsmeet Drakeriders riding the.
2. Player must use ;  [Hyldnir Harpoon]  quest item , to land on near by Photo- Drake  and kill the rider , and continue to do this until he kill 10 of them.
3. The problem is that there aren't any Photo-Drake's with Hyldsmeet Drakeriders riding them , flying around the Temple ofStorms . 
file:///B:/Wrath%20of%20the%20Lich%20King%203.3.5a%20(wod%20models)/Screenshots/WoWScrnShot_102517_004455.jpg
https://www.youtube.com/watch?v=oCcyIi7sqZE   -this is how it shoud be donne.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Zarjofgt** [comment]
file:///B:/Wrath%20of%20the%20Lich%20King%203.3.5a%20(wod%20models)/Screenshots/WoWScrnShot_102517_004455.jpg
![img_0032](https://user-images.githubusercontent.com/32937033/32001763-0c646ea6-b99b-11e7-9184-f4ae6504bf5b.JPG)

**Comment by TeridanDW** [comment]
Duplicate with #4349.
